scout/commands/setup/setup_scout.py

```diff
@@ -92,6 +92,14 @@ def abort_if_false(ctx, param, value):
         "phenotype_to_genes.txt"
     ),
 )
+@click.option(
+    "--hpo-phenotype-annotation",
+    type=click.Path(exists=True),
+    help=(
```

Comment:
Question: don't we need to update also the command line's `scout update hpo` with this new param? In the situation you downloaded the phenotype.hpoa and you want to use it when running the command scout update hpo?

Reply:
If I recall correctly (hey, it was last week after all) we only need the flag for Diseases, and then of course when we update everything. 

The ones that change is the ones that used to call `parse_hpo_diseases`, and that would include `scout update diseases` but not `scout update hpo`. 

But, this is a little tricky so I will reread it again - they have layered fallback to default files if one is missing, so its easy to miss..

Reply:
Ok, no, I stand by that.

update hpo (https://github.com/Clinical-Genomics/scout/blob/e48092a9aaa90f85545b1dd8363c7049799a73e2/scout/commands/update/hpo.py#L54)
will call `load_hpo_terms`, (https://github.com/Clinical-Genomics/scout/blob/e48092a9aaa90f85545b1dd8363c7049799a73e2/scout/load/hpo.py#L58) which never loads diseases. 

This is functionally in contrast to `load_hpo`, which is only called from the `setup` command, and will load both HPO and Diseases on its own rather than calling the two other functions, for some awkward historical reason or maybe optimisation reasons. But I'm not about to change that today. We could put harmonising that on that major list.